USACO DEC 09 GOLD COW TOLL PATH.md

```diff
@@ -0,0 +1,14 @@
+# USACO DEC 09 GOLD Cow Toll Path
+## Hồ Đức Hiếu
+
+## Problem Statement
+ Cho 1 đồ thị có n đỉnh (1<=n<= 250), và m cạnh (m <= 10 000). Mỗi cạnh và mỗi đỉnh của đồ thị đều có giá của nó. Giá của 1 chu trình được tính bằng tổng giá của các cạnh trong chu trình cộng với giá lớn nhất của các đỉnh nó đi qua (tính cả nút bắt đầu và nút kết thúc). Cho k truy vấn (k <= 10 000), với mỗi truy vấn i hãy in ra giá nhỏ nhất để đi từ đỉnh a_i đến b_i.
```

Comment:
Chỉnh thành thế này này
```text
Cho 1 đồ thị có n đỉnh (1 &le; n &le; 250), và m cạnh (m &le; 10 000). Mỗi cạnh và mỗi đỉnh của đồ thị đều có giá của nó. Giá của 1 chu trình được tính bằng tổng giá của các cạnh trong chu trình cộng với giá lớn nhất của các đỉnh nó đi qua (tính cả nút bắt đầu và nút kết thúc). Cho k truy vấn (k &le; 10 000), với mỗi truy vấn thứ i hãy in ra giá nhỏ nhất để đi từ đỉnh a<sub>i</sub> đến b<sub>i</sub>.
```
Và đây là cái nó hiện ra:
Cho 1 đồ thị có n đỉnh (1 &le; n &le; 250), và m cạnh (m &le; 10 000). Mỗi cạnh và mỗi đỉnh của đồ thị đều có giá của nó. Giá của 1 chu trình được tính bằng tổng giá của các cạnh trong chu trình cộng với giá lớn nhất của các đỉnh nó đi qua (tính cả nút bắt đầu và nút kết thúc). Cho k truy vấn (k &le; 10 000), với mỗi truy vấn thứ i hãy in ra giá nhỏ nhất để đi từ đỉnh a<sub>i</sub> đến b<sub>i</sub>.

**=> Hãy lợi dụng Markdown và HTML để nghịch lần sau =)**